app/views/dashboard/_hosts_in_build_mode_widget_host_list.html.erb

```diff
@@ -11,7 +11,7 @@
         <td class="ellipsis"><%= host_build_status_icon(host) %> <%= link_to host.name, host_path(host) %></td>
         <td class="hidden-tablet hidden-xs ellipsis"><%= host.owner %></td>
         <td class="hidden-tablet hidden-xs ellipsis"><%= build_duration(host) %></td>
-        <td class="hidden-tablet hidden-xs ellipsis"><%= host.token ? date_time_relative_value(host.token.expires) : _('N/A') %></td>
+        <td class="hidden-tablet hidden-xs ellipsis"><%= host.token ? date_time_relative(host.token.expires) : _('N/A') %></td>
```

Comment:
This should also work:
```suggestion
        <td class="hidden-tablet hidden-xs ellipsis"><%= date_time_relative(host.token&.expires) %></td>
```
Since the react component already get a `'N/A'` default value:
https://github.com/theforeman/foreman/blob/16b27a08a2e632c5a9c8cbf8eb8c84cf45ec2770/app/helpers/application_helper.rb#L55-L59

Reply:
why do I need '&'?

Reply:
in case the token is `nil` - `&.` is the safe navigation operator, it is basically short-hand for`host.token && host.token.expires`. Otherwise you would get `NoMethodError: undefined method `expires' for nil:NilClass` when the token is `nil`.

Reply:
great! thanks